client/src/App.jsx

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import './App.css';
-
+import NavBar from './components/UI/NavBar';
 import PacmanGame from './components/PacmanGame/PacmanGame';
 
 const App = () => (
-  <PacmanGame width={500} numberofCells={50} />
+  <div>
+    <NavBar isLoggedIn />
+    <div className="container">
```

Comment:
Is game can be played? even when the user is not logged in?

Reply:
yes

Reply:
If the user want to save his score and track his record, only then the user is required to Login